web/viewer.html

```diff
@@ -255,82 +255,82 @@
                 <span data-l10n-id="pdfjs-cursor-hand-tool-button-label">Hand Tool</span>
               </button>
             </div>
 
             <div class="horizontalToolbarSeparator"></div>
 
             <div id="scrollModeButtons" role="radiogroup">
-              <button id="scrollPage" class="toolbarButton" type="button" title="Use Page Scrolling" tabindex="62" data-l10n-id="pdfjs-scroll-page-button" role="radio" aria-checked="false">
+              <button id="scrollPage" class="toolbarButton labeled" type="button" title="Use Page Scrolling" tabindex="62" data-l10n-id="pdfjs-scroll-page-button" role="radio" aria-checked="false">
                 <span data-l10n-id="pdfjs-scroll-page-button-label">Page Scrolling</span>
               </button>
-              <button id="scrollVertical" class="toolbarButton toggled" type="button" title="Use Vertical Scrolling" tabindex="63" data-l10n-id="pdfjs-scroll-vertical-button" role="radio" aria-checked="true">
+              <button id="scrollVertical" class="toolbarButton labeled toggled" type="button" title="Use Vertical Scrolling" tabindex="63" data-l10n-id="pdfjs-scroll-vertical-button" role="radio" aria-checked="true">
                 <span data-l10n-id="pdfjs-scroll-vertical-button-label" >Vertical Scrolling</span>
               </button>
-              <button id="scrollHorizontal" class="toolbarButton" type="button" title="Use Horizontal Scrolling" tabindex="64" data-l10n-id="pdfjs-scroll-horizontal-button" role="radio" aria-checked="false">
+              <button id="scrollHorizontal" class="toolbarButton labeled" type="button" title="Use Horizontal Scrolling" tabindex="64" data-l10n-id="pdfjs-scroll-horizontal-button" role="radio" aria-checked="false">
                 <span data-l10n-id="pdfjs-scroll-horizontal-button-label">Horizontal Scrolling</span>
               </button>
-              <button id="scrollWrapped" class="toolbarButton" type="button" title="Use Wrapped Scrolling" tabindex="65" data-l10n-id="pdfjs-scroll-wrapped-button" role="radio" aria-checked="false">
+              <button id="scrollWrapped" class="toolbarButton labeled" type="button" title="Use Wrapped Scrolling" tabindex="65" data-l10n-id="pdfjs-scroll-wrapped-button" role="radio" aria-checked="false">
                 <span data-l10n-id="pdfjs-scroll-wrapped-button-label">Wrapped Scrolling</span>
               </button>
             </div>
 
             <div class="horizontalToolbarSeparator"></div>
 
             <div id="spreadModeButtons" role="radiogroup">
-              <button id="spreadNone" class="toolbarButton toggled" type="button" title="Do not join page spreads" tabindex="66" data-l10n-id="pdfjs-spread-none-button" role="radio" aria-checked="true">
+              <button id="spreadNone" class="toolbarButton labeled toggled" type="button" title="Do not join page spreads" tabindex="66" data-l10n-id="pdfjs-spread-none-button" role="radio" aria-checked="true">
```

Comment:
In #18385 the class was called `menuItem` instead of `labeled`, but I figured that `labeled` would be better because it matches the existing `toggled` and `hidden` extension classes more closely and it indicates more clearly what is different compared to regular toolbar buttons (which lack a label).

Sorry for not having noticed this regression in my earlier tests!